src/sort-filter-bar/sort-filter-bar.ts

```diff
@@ -204,122 +261,112 @@ export class SortFilterBar
-        id="desktop-sort-selector"
+      <div
+        id="desktop-sort-container"
         class=${this.mobileSelectorVisible ? 'hidden' : 'visible'}
       >
-        <li id="sort-by-text">Sort By</li>
-        <li>
-          ${this.showRelevance
-            ? this.getSortDisplayOption(SortField.relevance)
-            : nothing}
-        </li>
-        <li>
-          ${this.getSortDisplayOption(SortField.weeklyview, {
-            clickEvent: () => {
-              if (!this.viewOptionSelected)
-                this.setSelectedSort(SortField.weeklyview);
-              this.viewSortSelectorVisible = !this.viewSortSelectorVisible;
-              this.dateSortSelectorVisible = false;
-              this.alphaSelectorVisible = null;
-              this.selectedTitleFilter = null;
-              this.selectedCreatorFilter = null;
-              this.emitTitleLetterChangedEvent();
-              this.emitCreatorLetterChangedEvent();
-            },
-            displayName: html`${this.viewSortField}`,
-            isSelected: () => this.viewOptionSelected,
-          })}
-        </li>
-        <li>
-          ${this.getSortDisplayOption(SortField.title, {
-            clickEvent: () => {
-              this.alphaSelectorVisible = 'title';
-              this.selectedCreatorFilter = null;
-              this.dateSortSelectorVisible = false;
-              this.viewSortSelectorVisible = false;
-              this.setSelectedSort(SortField.title);
-              this.emitCreatorLetterChangedEvent();
-            },
-          })}
-        </li>
-        <li>
-          ${this.getSortDisplayOption(SortField.date, {
-            clickEvent: () => {
-              if (!this.dateOptionSelected)
-                this.setSelectedSort(SortField.date);
-              this.dateSortSelectorVisible = !this.dateSortSelectorVisible;
-              this.viewSortSelectorVisible = false;
-              this.alphaSelectorVisible = null;
-              this.selectedTitleFilter = null;
-              this.selectedCreatorFilter = null;
-              this.emitTitleLetterChangedEvent();
-              this.emitCreatorLetterChangedEvent();
-            },
-            displayName: html`${this.dateSortField}`,
-            isSelected: () => this.dateOptionSelected,
-          })}
-        </li>
-        <li>
-          ${this.getSortDisplayOption(SortField.creator, {
-            clickEvent: () => {
-              this.alphaSelectorVisible = 'creator';
-              this.selectedTitleFilter = null;
-              this.dateSortSelectorVisible = false;
-              this.setSelectedSort(SortField.creator);
-              this.emitTitleLetterChangedEvent();
-            },
-          })}
-        </li>
-      </ul>
+        <ul id="desktop-sort-selector">
+          <li>
+            ${this.showRelevance
+              ? this.getSortDisplayOption(SortField.relevance)
+              : nothing}
+          </li>
+          <li>${this.viewsDropdownTemplate}</li>
```

Comment:
Changed views/date to use dropdown components instead of the old "pill" selector popups. I've tried to keep things readable by splitting out the dropdown templates below.